code/game/objects/items/storage/fancy.dm

```diff
@@ -44,6 +46,11 @@
 	fancy_open = !fancy_open
 	update_icon()
 	. = ..()
+	if(!contents.len)
```

Comment:
Why don't you have a message when closing the cardboard if it contains stuff? You did that for the crayon one

Reply:
Because the fancy boxes all open and close, so the attack-self here should just toggle the box open/close instead of spamming the user about trying to break it down.